### PR TITLE
Add support for audio speed controller in keyboard navigation and TTS in Text Audio addon, re #8425

### PR DESCRIPTION
**Pull request by msawoniuk**
Commit also includes:
* Update reading so that speech texts are no longer read with lang Atrrybute
* Change default texts of TTS
* Add support for enter button to read content
* Add support for exit button to stop audio

---------------------------------------------------------------------------------------------

[Link do zadania](https://learneticsa.assembla.com/spaces/lorepo/tickets/8425/details)
[Link do wersji testowej](https://test-8425-dot-mauthor-dev.ew.r.appspot.com/)
[Link do lekcji testowej](https://test-8425-dot-mauthor-dev.ew.r.appspot.com/embed/5154879049826304)

### CODE REVIEW

**Review by wkarasSolwit — CHANGES_REQUESTED** [review]
Wymagane poprawki w dokumentacji

File: addons/TextAudio/documentation.md [hunk 1]
```diff
@@ -53,90 +43,118 @@ It's also important that audio files have <strong>constant bitrate</strong> beca
...
-		</ul>
+                </ul>
 		</td>
     </tr>
-<tr>
-        <td>onEnd</td>
+    <tr>
+        <td>mp3</td>
+        <td>
+            This property serves for uploading an audio file in mp3 format into the module. It's needed for Internet Explorer.
```
[the remark this answers is not in the document]
Comment (msawoniuk) [reply addons/TextAudio/documentation.md:55]: Usunięte zostało "It's needed for Internet Explorer."

File: addons/TextAudio/documentation.md [hunk 2]
```diff
@@ -53,90 +43,118 @@ It's also important that audio files have <strong>constant bitrate</strong> beca
...
+            Setting up this property is only needed when On Text Click Behavior is set to "Play the interval from vocabulary file".
+            <p><em>This property allows online resources. <a href="/doc/page/Online-resources">Find out more »</a></em></p>
+        </td>
+    </tr>
+    <tr>
+        <td>Vocabulary ogg</td>
+        <td>
+            This property serves for uploading a vocabulary audio file in ogg format into the module. It's needed for all common browsers (exclude Internet Explorer and Safari).
```
[the remark this answers is not in the document]
Comment (msawoniuk) [reply addons/TextAudio/documentation.md:85]: Usunięte zostało "It's needed for all common browsers (exclude Internet Explorer and Safari)."

**Review by wkarasSolwit — APPROVED** [review]
OK

**Pull request merged**